ramls/inventory/location.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "A (shelf) location, the forth-level location unit below institution, campus, and library.",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "id of this (shelf) location record as UUID.",
+      "type": "string"
+    },
+    "name": {
+      "description": "Name of the (shelf) location",
+      "type": "string"
+    },
+    "code": {
+      "description": "Code of the (shelf) location, usually an abbreviation of the name.",
+      "type": "string"
+    },
+    "description": {
+      "description": "Description of the (shelf) location.",
+      "type": "string"
+    },
+    "discoveryDisplayName": {
+      "description": "Name of the (shelf) location to be shown in the discovery.",
+      "type": "string"
+    },
+    "isActive": {
+      "description": "Whether this (shelf) location is active. Inactive (shelf) locations can no longer been used.",
+      "type": "boolean"
+    },
+    "institutionId": {
+      "description": "The UUID of the institution, the first-level location unit, this (shelf) location belongs to.",
+      "type": "string"
+    },
+    "institution": {
+      "type": "object",
+      "folio:$ref": "locinst.json",
+      "readonly": true,
+      "folio:isVirtual": true,
+      "folio:linkBase": "location-units/institutions",
+      "folio:linkFromField": "institutionId",
+      "folio:linkToField": "id",
+      "folio:includedElement": "locinsts.0"
+    },
+    "campusId": {
+      "description": "The UUID of the campus, the second-level location unit, this (shelf) location belongs to.",
+      "type": "string"
+    },
+    "campus": {
+      "type": "object",
+      "folio:$ref": "loccamp.json",
+      "readonly": true,
+      "folio:isVirtual": true,
+      "folio:linkBase": "location-units/campuses",
+      "folio:linkFromField": "campusId",
+      "folio:linkToField": "id",
+      "folio:includedElement": "loccamps.0"
+    },
+    "libraryId": {
+      "description": "The UUID of the library, the third-level location unit, this (shelf) location belongs to.",
+      "type": "string"
+    },
+    "library": {
+      "type": "object",
+      "folio:$ref": "locinst.json",
+      "readonly": true,
+      "folio:isVirtual": true,
+      "folio:linkBase": "location-units/libraries",
+      "folio:linkFromField": "libraryId",
+      "folio:linkToField": "id",
+      "folio:includedElement": "loclibs.0"
+    },
+    "details": {
+      "description": "Details about this (shelf) location.",
+      "type": "object"
+    },
+    "primaryServicePoint": {
+      "description": "The UUID of the primary service point of this (shelf) location.",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
```

Comment:
It seems like we already have UUID schema for validation. Why do we need to use this pattern?